src/settings/settings.go

```diff
@@ -19,13 +19,17 @@ type Settings struct {
 	RuntimeSubdirectory        string `envconfig:"RUNTIME_SUBDIRECTORY"`
 	RuntimeIgnoreDotFiles      bool   `envconfig:"RUNTIME_IGNOREDOTFILES" default:"false"`
 	LogLevel                   string `envconfig:"LOG_LEVEL" default:"WARN"`
-	RedisSocketType            string `envconfig:"REDIS_SOCKET_TYPE" default:"unix"`
-	RedisUrl                   string `envconfig:"REDIS_URL" default:"/var/run/nutcracker/ratelimit.sock"`
+	RedisSocketType            string `envconfig:"REDIS_SOCKET_TYPE" default:"tcp"`
+	RedisUrl                   string `envconfig:"REDIS_URL" default:"127.0.0.1:6379"`
 	RedisPoolSize              int    `envconfig:"REDIS_POOL_SIZE" default:"10"`
+	RedisAuth                  string `envconfig:"REDIS_AUTH"`
```

Comment:
Do we get an empty string as a default if we don't specify a default? Should an empty string be the default for this setting?

Reply:
Added a default empty string value for visibility